### DISCUSSION TITLE
Web font not loading

### DISCUSSION THREAD

**Discussion by kyungnapark** [thegetty/quire#188] [Get Help] [answered]
On the site, I'm not sure why Helvetica loads instead of the default IBM Plex Sans Condensed font, which is what I would like it to be. 

Any help would be greatly appreciated!

(On the site)
![Screen Shot 2021-09-23 at 7 20 07 PM](https://user-images.githubusercontent.com/87673056/134608556-773fcfc7-c1e8-4445-b029-85e205388748.png)

(On local host)
![Screen Shot 2021-09-23 at 7 21 00 PM](https://user-images.githubusercontent.com/87673056/134608643-57995a4e-9589-4d72-b28f-782a5266a10e.png)

**Comment by kyungnapark** [comment]
The font works on local but not on the server, and these are the error messages that I get:
![Screen Shot 2021-09-26 at 7 44 17 PM](https://user-images.githubusercontent.com/87673056/134838101-84d83286-897b-4ae6-818c-5d03e77b42a3.png)

**Reply by kyungnapark** [reply]
Think I might be getting somewhere with this site, as the solution involves resolving a not found error with .woff or woff2 fonts, but I got stuck at finding the web.config file.

https://hotcakescommerce.zendesk.com/hc/en-us/articles/210926903-HTTP-404-Not-Found-Error-with-woff-or-woff2-Font-Files

**Comment by geealbers** [comment, marked as answer]
Jenny, are you still having this issue? I think it may be due to your deploy setup and that the paths to the fonts are not what they need to be. @halfempty should be able to help with the specifics because I believe he's used this deploy setup before, but you'll need to look at the font paths in the webpack configuration file, `themes/default/webpack/webpack.config.prod.js`, and maybe also at the `baseURL` in your `config.yml` file and adjust those according to the URL where your site will be served from. It's like you might do if you were deploying to GitHub pages: https://quire.getty.edu/documentation/site-deploy/#github-pages. By adjusting the values in a way similar to it's suggested in those instructions, you should be able to get the font paths to resolve properly. I just can't help with the specifics of exactly what values to put where since I'm not familiar with the setup or the URLs. Hope this leads you in the right direction though!

**Reply by Erin-Cecele** [reply]
@pancaketents, checking in to see if you and @halfempty were able to resolve this issue as well.

**Reply by kyungnapark** [reply]
still have not resolved the issue :/ 
looked at this exchange which involved a similar setup (building the site from a subfolder) but changing like 92 to site/fonts/ has not worked
![image](https://user-images.githubusercontent.com/87673056/137074629-7689f0c1-9bd6-40bd-b7c9-fcb8c5c238b6.png)

**Reply by andreas152** [reply]
Hello,

just dropped by as the new newsletter on Quire arrived in my inbox.

I am not into this topic as an expert, but while 404 (not found) was in the list presented here, I checked for similar problems and hit the following, which might be helpful as there the problem was solved.

https://stackoverflow.com/questions/22317206/failed-to-load-resource-the-server-responded-with-a-status-of-404-not-found

Maybe all or parts of the argument can be used or be part of an answer here too.

**Reply by kyungnapark** [reply]
@geealbers your fix worked after all, I just misunderstood part of it and typed "site" literally instead of the name of the relevant folder. Thanks!
